### PR TITLE
Added pom  property to include test source directory to checkstyle. F…

### PR DESCRIPTION
**Pull request by MaksymVakuliuk**
…ixed code style of tests.

### CODE REVIEW

File: src/main/java/com/spring/boot/project/demo/controllers/InjectController.java [hunk 1]
```diff
@@ -23,7 +24,8 @@ public void init() {
         insertRoles();
     }
 
-    @RequestMapping("/injectreviewstodb")
+    @ApiOperation(value = "Inserting data from data to DB.")
+    @GetMapping("/injectreviewstodb")
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/boot/project/demo/controllers/InjectController.java]: it's hard to read it 
https://restfulapi.net/resource-naming/

File: src/test/java/com/spring/boot/project/demo/service/impl/ReviewServiceImplTest.java [hunk 2]
```diff
@@ -22,13 +25,11 @@
...
 import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.boot.test.context.SpringBootTest;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
 
 @RunWith(MockitoJUnitRunner.class)
 @SpringBootTest
 public class ReviewServiceImplTest {
-    private final List<String> EXPECTED_MOST_USED_WORDS = List.of("review", "text");
+    private static final List<String> EXPECTED_MOST_USED_WORDS = List.of("review", "text");
```
Comment (RomanDubovskyi) [inline src/test/java/com/spring/boot/project/demo/service/impl/ReviewServiceImplTest.java]: it's not a constant, let's change naming 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

**Review by RomanDubovskyi — COMMENTED** [review]
good, added minor comments

File: src/test/java/com/spring/boot/project/demo/service/impl/ReviewServiceImplTest.java (hunk 2 above)
Comment (MaksymVakuliuk) [reply src/test/java/com/spring/boot/project/demo/service/impl/ReviewServiceImplTest.java]: Am I need to change name or change List to ImmutableList? Are we need use this field as constant or can we use this just as variable?

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**